packages/react-popover/src/components/Popover/Popover.types.ts

```diff
@@ -76,6 +76,10 @@ export type PopoverState = PopoverCommons &
      * Callback to open/close the Popover
      */
     setOpen: (e: OpenPopoverEvents, open: boolean) => void;
+    /**
+     * Callback to toggle the open state of the Popover
+     */
+    toggleOpen: (e: OpenPopoverEvents) => void;
```

Comment:
New callback, used to avoid reading the `open` state in children components because of context selector issue